// Exception handling example 1

#include<iostream>
using namespace std;
int main()
{	
	int num,ans;
	num = 10;
	int b = 0;
	ans = num/b;
	cout << ans << endl;
	return 0;
}
